python/lsst/pipe/base/argumentParser.py

```diff
@@ -239,9 +239,10 @@ def __init__(self, name, usage = "%(prog)s input [options]", **kwargs):
             help="config override(s), e.g. -c foo=newfoo bar.baz=3", metavar="NAME=VALUE")
         self.add_argument("-C", "--configfile", dest="configfile", nargs="*", action=ConfigFileAction,
             help="config override file(s)")
-        self.add_argument("-L", "--loglevel", help="logging level")
-        self.add_argument("-T", "--trace", nargs="*", action=TraceLevelAction,
-            help="trace level for component", metavar="COMPONENT=LEVEL")
+        self.add_argument("-L", "--loglevel", nargs="*", action=LogLevelAction,
+            help="logging level for component; supported levels are [debug|warn|info|fatal] or an integer; "
+                "use -n for trace level n",
+            metavar="LEVEL|COMPONENT=LEVEL")
```

Comment:
It was not immediately obvious to me that the reference to `-n` meant `--loglevel -n` --- it looks like another argument name.